jac-uk/admin
# Feature/2424 Support Reset of Character Report

## What's included?
Closes #2424 

Related PR: [digital-platform: Feature/admin 2424 Support Reset of Character Report #1123](https://github.com/jac-uk/digital-platform/pull/1123)

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
Example exercise: [Ryan test 1 (Character issues)](https://jac-admin-develop--pr2459-feature-2424-support-mc5hdyo3.web.app/exercise/IbeEVi4Vtrpv9eVFzhZq/reports/character-issues)

Note: The "Reset" button is only visible for `Digital Super User`.

Steps:
1. Go to the Character Issues report and hit the Reset button.
2. Check if the data is repopulated from applications.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Demo:

https://github.com/jac-uk/admin/assets/79906532/a7ad1360-dc27-4b39-928e-ad9c63c8ecf3

## Related permissions
New permission: `canResetCharacterIssuesReport`

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
